pkg/ottl/expression.go

```diff
@@ -27,14 +27,19 @@ func (e Expr[K]) Eval(ctx context.Context, tCtx K) (any, error) {
 	return e.exprFunc(ctx, tCtx)
 }
 
+// Getter is a interface that returns value of type any.
 type Getter[K any] interface {
+	// Get retrieves a value of type Any.
```

Comment:
Please also mention that it returns an error if it has trouble getting the value.